tests/ParserTest.elm

```diff
@@ -558,53 +573,132 @@ expr =
 
                 -- https://github.com/elm/compiler/blob/dcbe51fa22879f83b5d94642e117440cb5249bb1/compiler/src/Parse/String.hs#L279-L285
                 , ( "escape n"
-                  , "'\\n'"
+                  , singleQuote "\\n"
                   , Ok (Literal (Char '\n'))
                   )
                 , ( "escape r"
-                  , "'\\r'"
+                  , singleQuote "\\r"
                   , Ok (Literal (Char '\u{000D}'))
                   )
                 , ( "escape t"
-                  , "'\\t'"
+                  , singleQuote "\\t"
                   , Ok (Literal (Char '\t'))
                   )
                 , ( "double quote"
-                  , "'\\\"'"
+                  , singleQuote "\\\""
```

Comment:
now that I look at it, I also wonder if this is right.

from the user perspective, `'"'`
from the parser input perspective, `"'\"'"`, which is `singleQuote "\""`
from the parser output perspective, `Char '"'`

WDYT?

Reply:
That's correct. Did I make a typo?

Reply:
Yeah, if I see correctly you do `singleQuote "\\\""` instead of `singleQuote "\""`